table/tables/state_remote.go

```diff
@@ -282,7 +290,7 @@ func waitForLeaseExpire(oldReadLease, now uint64) time.Duration {
 func (h *stateRemoteHandle) RenewReadLease(ctx context.Context, tid int64, oldLocalLease, newValue uint64) (uint64, error) {
 	var newLease uint64
 	err := h.runInTxn(ctx, false, func(ctx context.Context, now uint64) error {
-		lockType, remoteLease, _, err := h.loadRow(ctx, tid, false)
+		lockType, remoteLease, _, err := h.loadRow(ctx, tid, true)
```

Comment:
Why? the pessimistic parameter passed to `runInTxn` is false, so set forUpdate to true is useless..

Reply:
I think I can reset here ... this is a random change during debugging